<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ActionBox Example</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
MainWindow {
    background-color: white;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QSint::ActionPanel" name="ActionPanel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSint::ActionGroup" name="ActionGroup1" native="true">
     <property name="expandable" stdset="0">
      <bool>true</bool>
     </property>
     <property name="header" stdset="0">
      <bool>true</bool>
     </property>
     <property name="headerText" stdset="0">
      <string>Choose Scheme</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rbDefaultScheme">
        <property name="text">
         <string>Default</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbXPBlueScheme">
        <property name="text">
         <string>XP Blue</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbXPBlue2Scheme">
        <property name="text">
         <string>XP Blue 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbVistaScheme">
        <property name="text">
         <string>Vista</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbMacScheme">
        <property name="text">
         <string>MacOS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbAndroidScheme">
        <property name="text">
         <string>Android</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>57</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/filenew.png</normaloff>:/res/filenew.png</iconset>
   </property>
   <property name="text">
    <string>Create new file</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/fileopen.png</normaloff>:/res/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Load a file</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/filesave.png</normaloff>:/res/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save current file</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/fileprint.png</normaloff>:/res/fileprint.png</iconset>
   </property>
   <property name="text">
    <string>Print file contents</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSint::ActionPanel</class>
   <extends>QFrame</extends>
   <header>QSintActions</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QSint::ActionGroup</class>
   <extends>QWidget</extends>
   <header>QSintActions</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
